rand_distr/src/skew_normal.rs

```diff
@@ -40,9 +40,9 @@ use rand::Rng;
 /// [skew normal distribution]: https://en.wikipedia.org/wiki/Skew_normal_distribution
 /// [`Normal`]: struct.Normal.html
 /// [A Method to Simulate the Skew Normal Distribution]:
-/// 	Ghorbanzadeh, D. , Jaupi, L. and Durand, P. (2014)
-/// 	[A Method to Simulate the Skew Normal Distribution](https://dx.doi.org/10.4236/am.2014.513201).
-/// 	Applied Mathematics, 5, 2073-2076.
+///     Ghorbanzadeh, D. , Jaupi, L. and Durand, P. (2014)
+///     [A Method to Simulate the Skew Normal Distribution](https://dx.doi.org/10.4236/am.2014.513201).
+///     Applied Mathematics, 5, 2073-2076.
```

Comment:
This kind of reference works?

Reply:
I did not check the `cargo doc` output, so I'm not sure.

Reply:
Unfortunately, it doesn't:
![grafik](https://user-images.githubusercontent.com/33460/132955098-2c1db4cc-7be8-497d-8b5e-b2965683092a.png)